src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/PosBlockSignatureRule.cs

```diff
@@ -12,7 +12,7 @@ namespace Stratis.Bitcoin.Features.Consensus.Rules.CommonRules
     /// <summary>
     /// A rule that will validate the signature of a PoS block.
     /// </summary>
-    [PartialValidationRule(CanSkipValidation = true)]
+    [PartialValidationRule(CanSkipValidation = false)]
```

Comment:
Its false by default so you can just remove the setter, but this is fine also.

Reply:
Ah should this not be IntegrityValidator attribute? @Aprogiena

Reply:
Integrity validator should be block signature and Merkle tree root and mutation, so yes it should be

Reply:
It just means that another fix is required and Dan will probably take care of that.

Reply:
Yeah it means it will be executed in a different part of the process of downloading a block.